src/Sentry.Unity.Editor/SentryWindow.cs

```diff
@@ -138,4 +156,20 @@ private void OnGUI()
             // EditorGUILayout.EndToggleGroup();
         }
     }
+
+    public readonly struct ValidationError
+    {
+        public readonly string PropertyName;

```

Comment:
10+ years of brainwashing that public fields, even of readonly are evil make me have a hard time looking at this. Probably never to be a problem as it's not a public API that would break consumers if we needed to encapsulate later but curious, why not throw in a getter?
 ```suggestion
        public readonly string PropertyName { get; };
```

Reply:
It's `readonly struct`. From my perspective, it doesn't need to have a getter.

I see that you have a very strong opinion about that. If you want, I'll change it to a getter.

Reply:
Oh I don't have a very strong opinion, I even merged the PR :)
I was trying to word it even in a joke (10 years of brainwashin).

I can't argue why add a getter to a readonly struct so lgtm